eva-lib/src/test/java/uk/ac/ebi/eva/lib/repository/VariantEntityRepositoryTest.java

```diff
@@ -339,6 +339,26 @@ public void testFindByRegionsAndComplexFiltersExcludeSingleRoot() {
         }
     }
 
+    @Test
+    public void testFindByRegionsAndComplexFiltersExcludeSourceEntries() {
+        List<Region> regions = new ArrayList<>();
+        regions.add(new Region("11", 183000, 183300));
+
+        List<String> exclude = new ArrayList<>();
+        exclude.add("files.attrs");
```

Comment:
Ok I see I made a mistake here. there's no "attrs" constant as changed in the previous one, because there's no filter by attrs. here we are using the "exclude" which has nothing to do with filters. I'll change this so that the exclude ones are retrieved from the converters instead.